src/lrstar/PG_Generate.cpp

```diff
@@ -1935,6 +1935,13 @@ static void instantiate_getopt(FILE *fp)
            "                                 */\n"
            "    }\n"
            "}\n"
+           "\n"
+           "    static void\n"
+           "    fatal_free_opt(int rc)\n"
```

Comment:
Please mark this with the NORETURN annotation.
It's defined via including lrstar_basic_defs.h.

Also, either just free(options.output), or put it in a full block.